tests/shopify_python/test_google_styleguide.py

```diff
@@ -198,3 +198,24 @@ def fnc():
         message = pylint.testutils.Message('complex-list-comp', node=bad_list_comp)
         with self.assertAddsMessages(message):
             self.walk(root)
+
+    def test_use_cond_exprs(self):
+        root_to_msg = astroid.builder.parse("""
+            if 1 < 2:
+                num = 1
+            else:
+                num = 2
+            """)
+        if_node = root_to_msg.body[0]
+        message = pylint.testutils.Message('cond-expr', node=if_node)
+        with self.assertAddsMessages(message):
+            self.walk(root_to_msg)
+
+        root_to_not_msg = astroid.builder.parse("""
+            if 1 < 2:
+                num = 1
+            else:
+                dum = 2
+            """)
+        with self.assertAddsMessages():
```

Comment:
If you inspect the base class for `TestGoogleStyleGuideChecker` (`CheckerTestCase`), you'll see there's a member function named `assertNoMessages`.